examples/app_unity_gain_voice/src/app_unity_gain_voice.xc

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2017, XMOS Ltd, All rights reserved
+#include <xs1.h>
+#include <xclib.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>  /* abs */
+#include <math.h>    /* log10 */
+
+#include "src.h"
+
+#define DEBUG_PRINT (0)
+
+#define MAX_ATTEN_DB (-20)
+#define NUM_OF_SAMPLES (100)
```

Comment:
I would set this to the number of taps per phase (24) times the number of phases (3) plus one.